examples/mistral-completion-example/mistral_completion_example.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/tmc/langchaingo/llms"
+	"github.com/tmc/langchaingo/llms/mistral"
+)
+
+func main() {
+	llm, err := mistral.New(mistral.WithAPIKey("API_KEY_GOES_HERE"))
```

Comment:
not blocking but I can't help but think that this comment could be laid out in a manner that would be more accessible -- could you add a comment either at the program/file level in a way that would help new folks figure out what's going on? Even better would be a docs-site addition along a blog post.

Reply:
Yes, that doc site is cool! Will add some docs over this week.

Reply:
Added some docs for now.